baselines/constraint/constraint.py

```diff
@@ -8,6 +8,42 @@
 
 
 class Constraint(DFA):
+    '''
+    Constraint represents a given constraint to place on the agent.
+       
+    Attributes
+    ==========
+    name : str
+        The name of the constraint
+    reg_ex : str?
+        A representation of the constraint
+        using regular expressions.
+    violation_reward : int
+        The reward signal for violating the
+        constraint. Used in reward shaping.
+    s_tl : ?
```

Comment:
state translation layer, should be a function which takes one argument (the state as returned by the environment that `ConstraintWrapper` is wrapping) and returns a token in the alphabet of the DFA

Reply:
I added a TODO for you to extrapolate what the data-type is for both a_tl and s_tl.